test/common.sh

```diff
@@ -12,11 +14,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-#!/usr/bin/env bash
-
 # common.sh has helper functions, and is sourced by the other scripts in this directory
 set -o errexit
 set -o nounset
 set -o pipefail
 
-PROJECT_ROOT=$(dirname "${BASH_SOURCE}")/..
+# shellcheck disable=SC2034
```

Comment:
You don't need to disable this.  You can set the file with a special shellcheck command. https://github.com/koalaman/shellcheck/wiki/Directive#source